feathr_project/feathr/_feature_registry.py

```diff
@@ -645,69 +649,84 @@ def save_to_feature_config(self, workspace_path: Path):
     @classmethod
     def save_to_feature_config_from_context(self, anchor_list, derived_feature_list, local_workspace_dir: Path):
         """Save feature definition within the workspace into HOCON feature config files from current context, rather than reading from python files"""
-        repo_definitions = self._extract_features_from_context(anchor_list, derived_feature_list, local_workspace_dir)
-        self._save_request_feature_config(repo_definitions, local_workspace_dir)
-        self._save_anchored_feature_config(repo_definitions, local_workspace_dir)
-        self._save_derived_feature_config(repo_definitions, local_workspace_dir)
+        repo_definitions = self._extract_features_from_context(
+            anchor_list, derived_feature_list, local_workspace_dir)
+        self._save_request_feature_config(
+            repo_definitions, local_workspace_dir)
+        self._save_anchored_feature_config(
+            repo_definitions, local_workspace_dir)
+        self._save_derived_feature_config(
+            repo_definitions, local_workspace_dir)
 
     @classmethod
     def _save_request_feature_config(self, repo_definitions: RepoDefinitions, local_workspace_dir="./"):
-        config_file_name = "feature_conf/auto_generated_request_features.conf"        
-        tm = Template("""
-                    // THIS FILE IS AUTO GENERATED. PLEASE DO NOT EDIT.
-                    anchors: {
-                        {% for anchor in feature_anchors %}
-                            {% if anchor.source.name == "PASSTHROUGH" %}
-                                {{anchor.to_feature_config()}}
-                            {% endif %}
-                        {% endfor %}
-                    }
-                    """)
-        
-        request_feature_configs = tm.render(feature_anchors=repo_definitions.feature_anchors)
+        config_file_name = "feature_conf/auto_generated_request_features.conf"
+        tm = Template(
+            """
+// THIS FILE IS AUTO GENERATED. PLEASE DO NOT EDIT.
```

Comment:
@jaymo001 @hangfei Are you OK with this?

Reply:
LGTM. Thanks a lot for the contribution.

Reply:
and for reference - the only change here was to adjust the indentation of the block quote - it was causing weird formatting in the generated .conf files that made them difficult to read as a person - this is certainly not a mission critical change